cpp/src/slice2cpp/Gen.cpp

```diff
@@ -896,6 +912,25 @@ Slice::Gen::validateMetadata(const UnitPtr& u)
     };
     knownMetadata.emplace("cpp:ice_print", std::move(icePrintInfo));
 
+    // "cpp:identifier"
+    MetadataInfo identifierInfo = {
+        .validOn =
+            {typeid(InterfaceDecl),
+             typeid(Operation),
+             typeid(ClassDecl),
+             typeid(Slice::Exception),
```

Comment:
Why is Exception qualified?

Reply:
Just to be clear that we're dealing with the Slice type, and not `Ice::Exception` or any other base exception type.

Is is strictly necessary? No. But I think it makes the code clearer at a glance.
And it seems like we generally qualify `Ice::Exception` and `Slice::Exception` if you `rg` for `::Exception\b`.

Reply:
ok